### ISSUE TITLE
reversed the logic to   isDraggable: false,

### ISSUE THREAD

**Issue by robertoltrocha** [DevOrbiter/drag_and_drop_gridview#3]
Hi,

I think you have reversed the logic.
When I put isDraggable as true, it is not possible to drag

```
 itemBuilder: (context, index) => DragItem(
            isDraggable: true,
            child:
            Card(

          elevation: 2,
          child: LayoutBuilder(
            builder: (context, costrains) {
              if (variableSet == 0) {
                height = costrains.maxHeight;
                width = costrains.maxWidth;
                variableSet++;
              }
              return GridTile(
                child: Image.network(
                  _imageUris[index],
                  fit: BoxFit.cover,
                  height: height / 1.5,
                  width: width / 1.5,
                ),
              );
            },
          ),
        )),
        itemCount: _imageUris.length,
```

**Comment by IshanJaiswal99** [comment]
Greetings, 
Thanks for bringing that to our attention, we are working on that and hope to resolve it within the next 24 hrs.

**Comment by debashisnandy** [comment]
Bug fixed with version **1.0.7**. Thank you for reporting this issue.
